engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -475,7 +475,7 @@ namespace dmGameSystem
         if (!component->m_RenderConstants)
             component->m_RenderConstants = dmGameSystem::CreateRenderConstants();
 
-        dmGameSystem::SetRenderConstant(component->m_RenderConstants, name_hash, value_index, element_index, var);
+        dmGameSystem::SetRenderConstant(component->m_RenderConstants, GetMaterial(component, component->m_Resource), name_hash, value_index, element_index, var);
```

Comment:
Reverting to how it was previously.

TODO: We should rework the Set/GetMaterialConstant and the callback mechanisms. As a first step, the callback should provide the material pointer, so that we don't have to get it twice!